packages/expo-modules-test-core/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { getAllExpoModulesInWorkingDirectory } from './getStructure';
+import { printModules } from './mockgen';
+
+const command = process.argv[2];
+
+if (command === 'mockgen') {
+  const modules = getAllExpoModulesInWorkingDirectory();
+  printModules(modules);
+} else if (command === 'getStucture') {
```

Comment:
kebab-case is more common for the CLI commands
```suggestion
} else if (command === 'get-mocks-structure') {
```